Kernel/VM/Space.h

```diff
@@ -63,6 +65,11 @@ class Space {
     Region* find_region_from_range(const Range&);
     Region* find_region_containing(const Range&);
 
+    Vector<Region*> find_regions_intersecting(const Range&);
+
+    template<typename Callback>
+    void for_each_region_intersecting(const Range&, Callback);
```

Comment:
I believe this cannot work, since the compiler can't see the template definition from here. And since it's not used anyway, let's delete it entirely.

(@tomuta suggested templates to generalize the duplicated code away. Now there is no duplication anymore, so there is no need for a template.)

Reply:
Done, should I squash it into its creation?